<html>
<head>
  <title>Evernote Export</title>
  <basefont face="Times New Roman" size="2" />
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
  <meta name="exporter-version" content="YXBJ Windows/601535 (zh-CN, DDL); Windows/6.3.0 (Win64); EDAMVersion=V2;"/>
  <style>
    body, td {
      font-family: Times New Roman;
      font-size: 10pt;
    }
  </style>
</head>
<body>
<a name="2250"/>

<div>
<span><div><div><span style="font-family: &quot;Times New Roman&quot;; font-weight: bold;"><font style="font-size: 12pt;">Figure and tables from the 3'UTR eQTL project</font></span></div><div><br/></div><div><span style="font-size: 10pt; font-family: &quot;Times New Roman&quot;;">PART I: Overall patterns of eQTL with foucs on 3'UTR eQTL</span></div><div><table style="border-collapse: collapse; min-width: 100%;"><colgroup><col style="width: 70px;"></col><col style="width: 109px;"></col><col style="width: 348px;"></col><col style="width: 143px;"></col><col style="width: 121px;"></col><col style="width: 204px;"></col><col style="width: 190px;"></col></colgroup><tbody><tr><td style="width: 70px; padding: 8px; border: 1px solid;"><div style="text-align: center;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Number</span></div></td><td style="width: 109px; padding: 8px; border: 1px solid;"><div style="text-align: center;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Title</span></div></td><td style="width: 348px; padding: 8px; border: 1px solid;"><div style="text-align: center;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Figure</span></div></td><td style="width: 143px; padding: 8px; border: 1px solid;"><div style="text-align: center;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Figure Legend</span></div></td><td style="width: 121px; padding: 8px; border: 1px solid;"><div style="text-align: center;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Conclusions</span></div></td><td style="width: 204px; padding: 8px; border: 1px solid;"><div style="text-align: center;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Data&amp;Methods</span></div></td><td style="width: 190px; padding: 8px; border: 1px solid;"><div style="text-align: center;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Notes</span></div></td></tr><tr><td style="width: 70px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">1</span></div></td><td style="width: 109px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Distribution of PIP score for eQTL-gene pairs in different genome regions</span></div></td><td style="width: 348px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.pip_regXtiss.pool_boxplot.pdf"><img src="Result-Figures_3UTR_eQTL_files/5389e0dbaedafc3ef02e4632231bd8be.png" alt="res.pip_regXtiss.pool_boxplot.pdf"></a></div></td><td style="width: 143px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">For each genome region, distribution of PIP score for eQTL-gene pairs across all 49 tissues was shown.</span></div></td><td style="width: 121px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">1, significant differences of PIP score distribution among different genome regions indicated by AOV test;</span></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">2, relatively higher PIP score for UTR3 than other regions.</span></div></td><td style="width: 204px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Genotype and gene expression data from GTeX project for 49 human tissues was used, and SNP-gene pair with PIP score predicted by DAPG method.</span></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Detail of methods and data please refer to #.</span></div></td><td style="width: 190px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">./res.pip_regXtiss.pool_boxplot.pdf</span></div></td></tr><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">2</span></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Counts and proportion of HP eQTL  across 49 tissues </span><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">in different genome regions</span></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.hp_eqtl_regXtissue_ct.pdf"><img src="Result-Figures_3UTR_eQTL_files/721027c33fec7dc1576fba5db9383178.png" alt="res.hp_eqtl_regXtissue_ct.pdf"></a></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Counts and proportion of HP eQTLs was shown as stacked barplot. Each bar for one tissue sample and  different colors for genome regions.</span></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">1, 3'UTR take up relatively small part;</span></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">2, high variability of the counts of HP eQTL in different tissues</span></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Genotype and gene expression data from GTeX project for 49 human tissues used, and SNP for any SNP-gene pair with PIP score &gt;=0.5 (predicted by DAPG method) was taken as HP eQTL.</span></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Detail of methods and data please refer to #.</span></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div style="text-align: justify;"><span style="font-size: 10pt;">./</span>res.hp_eqtl_regXtissue_ct.pdf</div><div style="text-align: justify;"><br/></div><div style="text-align: justify;"><span style="font-size: 10pt;">Double check proportion of HP eQTL in different gene regions:</span><br/></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Proportion of HP eQTL among all SNPs tested in whole blood tissue:</span></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Regions AllSNP  withPIP  Proportion(ordered)</span></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">5utr   26.6k   1.5k    0.057</span></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">ncRNA    160k  2.2k    0.014</span></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">CDS     52.8k   0.6k     0.012</span></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">3utr 110k  0.4k     0.003</span></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">intron  3301k  8k    0.002</span></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">intergenic      146753k  9.7k    0.00006</span></div><div style="text-align: justify;"><font face="Times New Roman" style="font-size: 10pt;"><br/></font></div></td></tr><tr><td style="width: 70px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">3</span></div></td><td style="width: 109px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Distribution of frequency of HP eQTLs observation across the 49 tissues for different genome regions.</span></div></td><td style="width: 348px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.HPeQTL_conservation_tissues_distr.pdf"><img src="Result-Figures_3UTR_eQTL_files/078fcbb748189ebc742f189006b6eb86.png" alt="res.HPeQTL_conservation_tissues_distr.pdf"></a></div></td><td style="width: 143px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Distribution of HP eQTL frequency for each genome region shown, and the Bin1-4 represent </span> <span style="letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; font-family: &quot;Times New Roman&quot;; font-size: 10pt; color: rgb(0, 0, 0); font-variant-caps: normal; font-variant-ligatures: normal;">frequency of HP eQTLs observation in the 49 tissues: </span><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;"> 1 (Bin1), 2-5(Bin2), 6-10(Bin3) and 11-49(Bin4). Colors for genome regions. </span></div></td><td style="width: 121px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">1, HP eQTL from within 3'UTR tend to be less conserved than codining region and 5'UTR ones, and more conserved than intron and intergenic ones.</span></div></td><td style="width: 204px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Number of tissues in which the eQTL-gene pair with PIP score &gt;=0.5 was calculated and the numbers were binned to 4 groups : 1 (Bin1), 2-5(Bin2), 6-10(Bin3) and 11-49(Bin4), then proportion of HP eQTL -gene pairs within each of the 4 bins was calculated for each genome region.</span></div></td><td style="width: 190px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">./res.HPeQTL_conservation_tissues_distr.pdf</span></div></td></tr><tr><td style="width: 70px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">4</span></div></td><td style="width: 109px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Estimates of true eQTL proportion in each genome region using pi1.</span></div></td><td style="width: 348px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.snp_gene_pi1_distr.pdf"><img src="Result-Figures_3UTR_eQTL_files/1cd3d5d57b811a325b1a31917cae645c.png" alt="res.snp_gene_pi1_distr.pdf"></a></div></td><td style="width: 143px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Distribution of pi1 across 18 tissues for each genome region was shown as boxplot</span></div></td><td style="width: 121px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">1, proportion of true eQTLs in 3'UTR is relatively high eventhough slightly lower than 5'UTR and promoter regions.</span></div></td><td style="width: 204px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">p values for all SNP-gene pairs association test was used as input for qvalue function in R, and pi0 estimated according to p value distribution, then the pi1: 1-pi0 was calculated for each genome region in each of 18 tissues.    </span></div></td><td style="width: 190px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">./res.snp_gene_pi1_distr.pdf</span></div></td></tr><tr><td style="width: 70px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">5</span></div></td><td style="width: 109px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Distribution estimate of eQTL overlap between tissues using pi1 estimates</span></div></td><td style="width: 348px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.tissue_comp_pi1_regions.pdf"><img src="Result-Figures_3UTR_eQTL_files/954a8e9cb3f90ca3c53732b8abc97231.png" alt="res.tissue_comp_pi1_regions.pdf"></a></div></td><td style="width: 143px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Distribution of estimated shared eQTLs between every  two of 18 tissues (153 comparisons in total) using pi1 method for different genome regions. Y-axis for pi1 and each boxplot for one genome region.</span></div></td><td style="width: 121px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">-</span></div></td><td style="width: 204px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">For any two tissues  out of all possible combinations of 18 tissues,  eQTL-gene pairs was selected based on PIP score &gt;=0.5 in one tissue and  pi1 was estimated based on their p values in another tissue. the pi1 was estimated for each of different genome regions seperately.     </span></div></td><td style="width: 190px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">./res.tissue_comp_pi1_regions.pdf</span></div></td></tr><tr><td style="width: 70px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">6</span></div></td><td style="width: 109px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">pi1 estimates of 3'UTR eQTL  between tissues pairs (1)</span></div><div style="text-align: justify;"><font face="Times New Roman" style="font-size: 10pt;"><br/></font></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Note: q value was used to selected significant eQTL in the first tissue tissue for test on second tissue</span></div></td><td style="width: 348px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.pi1_tissue_comp.3utr.heatmap.pdf"><img src="Result-Figures_3UTR_eQTL_files/8d30dae5b98c18f4e4d8928c792d8a9d.png" alt="res.pi1_tissue_comp.3utr.heatmap.pdf"></a></div></td><td style="width: 143px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Heatmap to show pi1 estimate of 3'UTR eQTL between any two tissue pairs of 18 tissues. the color density represent pi1 values and tissues was clustered using hclust in R.</span></div><div style="text-align: justify;"><font face="Times New Roman" style="font-size: 10pt;"><br/></font></div><div style="text-align: justify;"><font face="Times New Roman" style="font-size: 10pt;"><br/></font></div></td><td style="width: 121px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">1, Brain clustered together</span></div></td><td style="width: 204px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">For any two tissues  out of all possible combinations of 18 tissues,  3'UTR eQTL-gene pairs was selected based on q value &lt; 0.01 in one tissue and  pi1 was estimated based on their p values in another tissue. </span></div></td><td style="width: 190px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">./res.pi1_tissue_comp.3utr.heatmap.pdf</span></div></td></tr><tr><td style="width: 70px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">7</span></div></td><td style="width: 109px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">pi1 estimates of 3'UTR eQTL  between tissues pairs (1)</span></div><div style="text-align: justify;"><font face="Times New Roman" style="font-size: 10pt;"><br/></font></div><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Note: PIP score was used to selected significant eQTL in the first tissue tissue for test on second tissue</span></div></td><td style="width: 348px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.pi1_tissues_comp_via_HPeQTL.3utr.headtmap.pdf"><img src="Result-Figures_3UTR_eQTL_files/96170d3fea2baf0afe7a512f8d600c54.png" alt="res.pi1_tissues_comp_via_HPeQTL.3utr.headtmap.pdf"></a></div></td><td style="width: 143px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Heatmap to show pi1 estimate of 3'UTR eQTL between any two tissue pairs of 18 tissues. the color density represent pi1 values and tissues was clustered using hclust in R.</span></div></td><td style="width: 121px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">-</span></div></td><td style="width: 204px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">For any two tissues  out of all possible combinations of 18 tissues,  3'UTR eQTL-gene pairs was selected based on PIP score &gt;=0.5 in one tissue and  pi1 was estimated based on their p values in another tissue. </span></div></td><td style="width: 190px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">./res.pi1_tissues_comp_via_HPeQTL.3utr.headtmap.pdf</span></div></td></tr><tr><td style="width: 70px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">8</span></div></td><td style="width: 109px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Proportion of shared 3'UTR HP eQTL  between tissues pairs</span></div></td><td style="width: 348px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.hp_eqtl_overlap_between_tissues_3utr.heatmap.pdf"><img src="Result-Figures_3UTR_eQTL_files/447a2fa37b0df61745de0eeb1ef5c5b9.png" alt="res.hp_eqtl_overlap_between_tissues_3utr.heatmap.pdf"></a></div></td><td style="width: 143px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Heatmap to show shared 3'UTR HP eQTL between any two tissue pairs of 49 tissues. the color density represent overlap and tissues was clustered using hclust in R.</span></div></td><td style="width: 121px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">1, Same/similar tissues clustered together</span></div></td><td style="width: 204px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">For any two tissues  out of all possible combinations of 49 tissues,  proportion of shared 3'UTR HP eQTL-gene pairs was calculated as average of proportion of overlapped  3'UTR HP eQTLs among all 3'UTR HP eQTLs. </span></div></td><td style="width: 190px; padding: 8px; border: 1px solid;"><div style="text-align: justify;"><span style="font-family: &quot;Times New Roman&quot;; font-size: 10pt;">./res.hp_eqtl_overlap_between_tissues_3utr.heatmap.pdf</span></div></td></tr><tr><td style="width: 70px; padding: 8px; border: 1px solid;"><div>9</div></td><td style="width: 109px; padding: 8px; border: 1px solid;"><div>Consistence of eQTL effect on associated gene expression across tissues for HP eQTL and control SNPs</div></td><td style="width: 348px; padding: 8px; border: 1px solid;"><div><a href="Result-Figures_3UTR_eQTL_files/res.eQTL_incr_decr_slop_deviation.pdf"><img src="Result-Figures_3UTR_eQTL_files/7e69adec73da29bc9fa14d60de0fe476.png" alt="res.eQTL_incr_decr_slop_deviation.pdf"></a></div></td><td style="width: 143px; padding: 8px; border: 1px solid;"><div>Proportion of major eQTL effect (increase or decrease) on gene expression for HP eQLT and control SNPs across 18 tissues was shown.</div></td><td style="width: 121px; padding: 8px; border: 1px solid;"><div>1, HP eQTL tend to deviate from 0.5 very much compared against control SNP, which means more consistence of expression changing effect for HP eQTL;</div><div><br/></div><div>2,  the control should be around 0.5 as expectation, but there still could be some true eQTL in the control SNPs, so some deviation observed. </div></td><td style="width: 204px; padding: 8px; border: 1px solid;"><div>Regression coefficients between eQTL and associated gene expression was used to determine different changing effect of increase (&gt;0), decrease (&lt;0) when regression p &lt; 0.01 or uncertain effect when regression p &gt; 0.01 in each tissue, and then the number of  tissues with increase or decrease effects in the 18 tissues obtained, proportion for the major effect was calculated as Deviation or Consistence of changing effects here (Note: 0.5 was  random expectation).</div></td><td style="width: 190px; padding: 8px; border: 1px solid;"><div>./res.eQTL_incr_decr_slop_deviation.pdf</div><div><br/></div><div>TODO: should classified into different gene regions;</div><div>Control should be around 0.5</div></td></tr></tbody></table><div><font face="Times New Roman" style="font-size: 10pt;"><br/></font></div><div><font face="Times New Roman" style="font-size: 10pt;">PART II, functional evidences for 3'UTR eQTLs</font></div></div><table style="border-collapse: collapse; min-width: 100%;"><colgroup><col style="width: 70px;"></col><col style="width: 109px;"></col><col style="width: 348px;"></col><col style="width: 143px;"></col><col style="width: 121px;"></col><col style="width: 204px;"></col><col style="width: 190px;"></col></colgroup><tbody><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div style="text-align: justify;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Number</span></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div style="text-align: justify;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Title</span></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div style="text-align: justify;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Figure</span></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div style="text-align: justify;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Figure Legend</span></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div style="text-align: justify;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Conclusions</span></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div style="text-align: justify;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Data&amp;Methods</span></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div style="text-align: justify;"><span style="font-weight: bold; font-family: &quot;Times New Roman&quot;; font-size: 10pt;">Notes</span></div></td></tr><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div style="text-align: justify;">1</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div style="text-align: justify;">Enrichment of 3'UTR HP eQTL in functional elements</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.hp_eqtl_reg_enrichment_OR.pdf"><img src="Result-Figures_3UTR_eQTL_files/5fa7a157ae0426746e9c42787dec6033.png" alt="res.hp_eqtl_reg_enrichment_OR.pdf"></a></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div style="text-align: justify;">Odds ratio of 3'UTR HP eQTL enrichment within RBP binding regions, miRNA binding sites (seed match plus 10 bp flanking region), m6A sites (m6A and the 2 bp flanking region), polyadenylation signal sites was show by comparison agaist control SNPs. Each dot represent OR of each tissue, and the vertical line for 95% confidence interval. Red star above each line represent significance level of enrichment by fisher's exact test (* for &lt;0.05, ** for &lt;0.01 and *** for &lt; 0.001). </div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div style="text-align: justify;">1, significant enrichment of HP eQTL within RBP binding regions, m6A sites, polyadnylation signal sites, and slightly enrichment within miRNA binding sites observed; </div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div style="text-align: justify;">1, RBP binding regions was obtained from POSTAR database, and selected tissues of HEK293,HeLa, HepG2, K562 which contain binding sites for many more RBPs was used. binding regions &gt;50 bp truncated to be 50 bp, as a result on average # binding regions for each of  # RBPs analyzed here; Allele-specific RBP binding sites was downloade from published work by #, 7489 binding sites for 147 RBPs in 2 cell lines : HepG2 and K562 obtained and used here. </div><div style="text-align: justify;"><br/></div><div style="text-align: justify;">2, # miRNA binding sites for # miRNAs  from targetScan database (conserved predictions) downloaded, and 10 bp extended around the seed match used here for enrichment analysis;</div><div style="text-align: justify;"><br/></div><div style="text-align: justify;">3, 477453 m6A sites was obtained from RMBase database, 2bp flanking each m6a site that may affect the modification was analyzed here;</div><div style="text-align: justify;"><br/></div><div style="text-align: justify;">4, 35316 polyadenylaton signal sites in 3'UTR that may affect polyadenylation was obtained from the UTRdb.</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div style="text-align: justify;">./res.hp_eqtl_reg_enrichment_OR.pdf</div></td></tr><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div style="text-align: justify;">2</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div style="text-align: justify;">Enrichment of 3'UTR HP eQTL in RBP binding regions detected by eCLIP <br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.total_HP_eQTL_RBPB_OR_p_distr.pdf"><img src="Result-Figures_3UTR_eQTL_files/e6ac90023317158b746dca1fac21c3f0.png" alt="res.total_HP_eQTL_RBPB_OR_p_distr.pdf"></a><br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div style="text-align: justify;">OR (log2) and p values (- log10)  for 3'UTR HP eQTL (pooled across tissues) enrichment  within RBP binding regions was show. Each point for one RBP, x-axis for log2 OR and y-axis for - log10 p value. Names for top RBPs (top 10% for both p value and OR) were labeled.</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div style="text-align: justify;">most RBPs with enrichment signal show significant HP eQTL enrichment compared to control SNPs</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div style="text-align: justify;">eQTL with PIP score &gt;=0.5 in any of 49 tissues was pooled together as HP eQTL set and compared with simulated SNPs by SNPsnap as control. </div><div style="text-align: justify;"><br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div style="text-align: justify;">./res.total_HP_eQTL_RBPB_OR_p_distr.pdf</div></td></tr><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div style="text-align: justify;">3</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div style="text-align: justify;">3'UTR HP eQTL enrichment within RBP binding regions across 49 tissues</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.HP_eqtl_RBPB_enrich_bulbplot_sigRBPs.pdf"><img src="Result-Figures_3UTR_eQTL_files/ad6340bdff145434772dfba65ed61d90.png" alt="res.HP_eqtl_RBPB_enrich_bulbplot_sigRBPs.pdf"></a><br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div style="text-align: justify;">bulb plot to show  odds ratio (log2) and p value (-log10, fisher's exact test) for 3'UTR HP eQTL enrichment within binding regions for RBPs across 49 tissues, RBPs with pooled HP eQTL or control SNPs erichment within their binding regions (raw p value &lt; 0.01) was selected and shown here. here the dot size represent - log10 p value, and dot color intensity for log2 odds ratio, the rows (RBPs) and columns(Tissues) are clustered using hclust in R.</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div style="text-align: justify;">1, most RBP show over enrichment of 3'UTR HP eQTL within their binding regions compared with control</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div style="text-align: justify;">1, 3'UTR HP eQTL enrichment within binding region for each RBP in each tissue was tested agaist control SNPs (simulated by SNPsnap) using fisher's exaxt test, and  OR of HP eQTL vs control obtained;</div><div style="text-align: justify;">2, RBPs with their binding regions show significant enrichment of pooled HP eQTL with p value &lt; 0.01 by fisher's exact test was shown here. </div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div style="text-align: justify;">./res.HP_eqtl_RBPB_enrich_bulbplot_sigRBPs.pdf</div></td></tr><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div style="text-align: justify;">4</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div style="text-align: justify;">Enrichment of HP eQTL within m6A peaks across different genome regions for each tissue</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.hp_eqtl_m6A_enrich_reg_sameTiss_OR.pdf"><img src="Result-Figures_3UTR_eQTL_files/4316962ba1f7735b217188b2fc8a32a2.png" alt="res.hp_eqtl_m6A_enrich_reg_sameTiss_OR.pdf"></a><br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div style="text-align: justify;">Odds ratio for HP eQTL enrichment within each genome regions was ploted as dot with vertical line for 95% confidence interval. The dashed line indicates OR of 1, and red star was marked if the enrichmet is significant using fisher's exact test.  </div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div style="text-align: justify;">1, fisher's exact test was used to compare counts of HP eQTL within m6A peaks against control SNPs for each genome region seperately;</div><div style="text-align: justify;">2, m6A peaks for 8 human tissues was obtained from published work of Shan xiao (2019 NCB), and HP eQTL from matched tissues with m6A was used for enrichment test.</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div style="text-align: justify;">1, intron and intergenic HP eQTL show higher enrichment in m6A peaks than others, and 3'UTR show higher enrichment than other gene regions;</div><div style="text-align: justify;">2, the pattern is conserved across different tissues</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div style="text-align: justify;">./res.hp_eqtl_m6A_enrich_reg_sameTiss_OR.pdf</div></td></tr><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div style="text-align: justify;">5</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div style="text-align: justify;">Impact of 3'UTR HP eQTL variation on RNA secondary structue changes</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.d_RNAsnp.hp_control.pdf"><img src="Result-Figures_3UTR_eQTL_files/bc35c7f4d9e6763ccf0c1a604e02e3e4.png" alt="res.d_RNAsnp.hp_control.pdf"></a><br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div style="text-align: justify;">Boxplot to show distribution of RNA secondary strucutre changes using eucledian distance (reported by RNAsnp) between sequence containg reference and alternative alleles for pooled 3'UTR HP eQTLs (HP) and Control SNPs. AOV test p value for differeneces was shown;</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div style="text-align: justify;">1, 3'UTR HP eQTL tend to change RNA secondary structure more than control SNPs</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div style="text-align: justify;">1, RNAsnp (mode2) was used to predict the distance of RNA secondary strucutre between reference allele and alternate allele for each SNP, breifly, 200 bp flanking sequences for each SNP was extracted from transcripts and MFE structures compared for eucledian distance; </div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div style="text-align: justify;">./res.d_RNAsnp.hp_control.pdf</div></td></tr><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div style="text-align: justify;">6</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div style="text-align: justify;">Base pairing probability dostribution for 3'UTR HP eQTL and Control SNPs </div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.bp_prob_diff_abs_distr.pdf"><img src="Result-Figures_3UTR_eQTL_files/d32dfb096140ddcedbe1753abf28aded.png" alt="res.bp_prob_diff_abs_distr.pdf"></a><br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div style="text-align: justify;">Boxplot to show distribution of base pairing probability for the references allele of  3'UTR HP eQTLs (HP) and Control SNPs (reported by RNAfold). AOV test p value for differeneces was shown.<br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div style="text-align: justify;">1, reference allele of 3'UTR HP eQTL tend to base pair slightly more frequenty than control SNPs, which is consistent with the RNAsnp distance pattern.</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div style="text-align: justify;">1, RNA secondary structure ensembl for the reference allele and 200 flanking regions on transcripts was predicted by RNAfold software using default parameters, base pairing probability for the reference allele was summed up and compared. </div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div style="text-align: justify;">./res.bp_prob_diff_abs_distr.pdf</div></td></tr><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div style="text-align: justify;">7</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div style="text-align: justify;">Relationship between predicted RBP binding probability and gene expression association for 3'UTR HP eQTL across tissues</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.deepsea_RBPB_diff_vs_expression_slop.pdf"><img src="Result-Figures_3UTR_eQTL_files/43be21059af011bc7876ee57042882cd.png" alt="res.deepsea_RBPB_diff_vs_expression_slop.pdf"></a><br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div>Scatter plot of RBP binding probability changes between reference and alternate allele predicted by deepsea (x-axis) and regression coefficients with target gene expression across 18 tissues (y-axis) for  each 3'UTR HP eQTL was shown  for each RBP. One dot represent one SNP in each tissue. For each RBP, SNPs with predicted RBP binding probability differences  &gt;0.05 and absolute value of expression regression coefficient differences &gt; 0.05 was shown in each panel, the correlation coefficients and p value (pearson) was marked. </div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div style="text-align: justify;">1, BUD13, HNRNPK, U2AF1 and AGO show clear patten of correlation between RBP binding and expression changing trend;</div><div style="text-align: justify;"><br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div style="text-align: justify;">1, for each 3'UTR HP eQTL, the association with gene expression level was obtained from the GteX project, with correlation coefficients and p value reported acorss 18 tissues if data available;</div><div style="text-align: justify;">2, the RBP binding probability for reference allele and alternate allele for each HP eQTL was predicted by deepsea for # RBPs;</div><div style="text-align: justify;">3, HP eQTLs with RBP binding probability changes from reference to alternate alleles &gt; 0.05 was selected for the RBP, and their relationship with gene expression changes across all tissues was tested for each RBP seperately</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div style="text-align: justify;">./res.deepsea_RBPB_diff_vs_expression_slop.pdf</div></td></tr><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div style="text-align: justify;">8</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div style="text-align: justify;">Relationship between predicted RBP binding probability and gene expression association for 3'UTR HP eQTL across tissues<br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.motifbreak_effect_vs_expression_slop.pdf"><img src="Result-Figures_3UTR_eQTL_files/05c5c8b93595d5214b8e87a5716f0f65.png" alt="res.motifbreak_effect_vs_expression_slop.pdf"></a><br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div style="text-align: justify;">Scatter plot of RBP binding probability changes between reference and alternate allele predicted by motifbreak (x-axis) and regression coefficients with target gene expression across 18 tissues (y-axis) for  each 3'UTR HP eQTL was shown  for each RBP. One dot represent one SNP in each tissue. For each RBP, SNPs with predicted RBP binding probability differences  &gt;# and absolute value of expression regression coefficient differences &gt; # was shown in each panel, the correlation coefficients and p value (pearson) was marked. <br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div style="text-align: justify;">1, ELAVL4, PCBP1, TARDBP, SFRS7, and SRSF6 show clear patten of correlation between RBP binding and expression changing trend</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div style="box-sizing: border-box; -webkit-tap-highlight-color: transparent; margin: 0px; padding: 0px; color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-size: medium; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-align: justify;"><span style="box-sizing: border-box; -webkit-tap-highlight-color: transparent; color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-size: medium; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px;">1, for each 3'UTR HP eQTL, the association with gene expression level was obtained from the GteX project, with correlation coefficients and p value reported acorss 18 tissues if data available;</span></div><div style="box-sizing: border-box; -webkit-tap-highlight-color: transparent; margin: 0px; padding: 0px; color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-size: medium; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-align: justify;"><span style="box-sizing: border-box; -webkit-tap-highlight-color: transparent; color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-size: medium; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px;">2, the RBP binding probability for reference allele and alternate allele for each HP eQTL was predicted by motifbreak for # RBPs;</span></div><div style="text-align: justify;"><br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div style="text-align: justify;">./res.motifbreak_effect_vs_expression_slop.pdf</div></td></tr><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div style="text-align: justify;">9</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div style="text-align: justify;">Gene function annotation for RBP with 3'UTR HP eQTL enrichment</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/dat.hp_control.RBPB.enrich.pool.sig.geneanno.xls"><img src="Result-Figures_3UTR_eQTL_files/94324f2921905b130e386a6754ec4f6e.png" alt="dat.hp_control.RBPB.enrich.pool.sig.geneanno.xls"></a><br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div style="text-align: justify;">This table show statistics for top RBPs with significant 3'UTR HP eQTL enrichment (p&lt;0.01) along with gene function annotation from genecard database</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div style="text-align: justify;">1, out of the 31 RBPs, 15 related to splicing, 5 for polyadenylation,  3  show functions releated to m6A,and 2 for degradation</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div style="text-align: justify;">-</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div style="text-align: justify;"><br/></div></td></tr><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div style="text-align: justify;">10</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div style="text-align: justify;">RBPs with both 3'UTR HP eQTL enrichment and strong association between RBP binding and expression regression</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div style="text-align: justify;"><a href="Result-Figures_3UTR_eQTL_files/res.deepsea_RBPB_diff_vs_expression_slop.corr.p.HP_eQTL_enrich.xls"><img src="Result-Figures_3UTR_eQTL_files/b879435ed9f2f383eb41b7a36f77c682.png" alt="res.deepsea_RBPB_diff_vs_expression_slop.corr.p.HP_eQTL_enrich.xls"></a><br/></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div style="text-align: justify;">This table show selected RBPs for which HP eQTL enrichment detected using pooled 3'UTR HP eQTL , and strong association between the regression with gene expression and predicted RBP binding probability changes (by deepsea) from reference allele to alternate allele </div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div style="text-align: justify;">1, 8 intersting RBPs listed could be investigated further </div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div style="text-align: justify;">-</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div style="text-align: justify;"><br/></div></td></tr><tr><td style="border: 1px solid rgb(204, 204, 204); width: 70px; padding: 8px;"><div>11</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 109px; padding: 8px;"><div>OR of enrichment of HP eQTL within GWAS signals across genome regions</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 348px; padding: 8px;"><div><a href="Result-Figures_3UTR_eQTL_files/res.snp_gwas_generegions.xls"><img src="Result-Figures_3UTR_eQTL_files/35fdc4084ba4f7c027c7cc3945cbbef7.png" alt="res.snp_gwas_generegions.xls"></a></div></td><td style="border: 1px solid rgb(204, 204, 204); width: 143px; padding: 8px;"><div>This table show enrichment of pooled HP eQTL in different genome regions </div></td><td style="border: 1px solid rgb(204, 204, 204); width: 121px; padding: 8px;"><div>1, higher enrichment of intron and integenic HP eQTL in GWAS signals observed </div></td><td style="border: 1px solid rgb(204, 204, 204); width: 204px; padding: 8px;"><div>-</div></td><td style="border: 1px solid rgb(204, 204, 204); width: 190px; padding: 8px;"><div><br/></div></td></tr></tbody></table><div><br/></div><div><font face="Times New Roman" style="font-size: 10pt;"><br/></font></div><div><font face="Times New Roman" style="font-size: 10pt;"><br/></font></div><div><font face="Times New Roman" style="font-size: 10pt;"><br/></font></div><div><font face="Times New Roman" style="font-size: 10pt;"><br/></font></div><div><font face="Times New Roman" style="font-size: 10pt;"><br/></font></div><div><br/></div></div></span>
</div></body></html> 